### ISSUE TITLE
Issues with jaxlib

### ISSUE THREAD

**Issue by TrainerSnow** [painebenjamin/app.enfugue.ai#96]
When clicking the `Enfugue` button in the UI to generate the image, I get the following error message ~7 seconds afterward:
```
This version of jaxlib was built using AVX2 instructions, which your CPU and/or operating system do not support. You may be able work around this issue by building jaxlib from source.
```
I'm on windows and used the windows zip from the releases tab.

**Comment by painebenjamin** [comment]
Hello @TrainerSnow,

Thank you for the bug report. To be clear, this message appears as an alter in your UI and you cannot proceed, correct? I only ask because I've seen this message and it has been innocuous before, but it's very possible that it is breaking your installation.

I am currently building v0.3 which has modified dependencies, so I can take another look at the jaxlib version. Stay tuned, release is expected on Monday.

Thanks again!

**Comment by TrainerSnow** [comment]
Yep, it appears in a red box in the bottom right corner. I'll let you know if 0.3 fixes the issue 👍

**Comment by painebenjamin** [comment]
Hello @TrainerSnow!

The dependency on a specific version of Jaxlib has been removed in v0.3. There's a possibility that it was merely covering up a different issue that will now rear it's head, but 0.3 should at least get you past this part.

It's up now on the releases page: https://github.com/painebenjamin/app.enfugue.ai/releases

**Comment by coreshock** [comment]
I'm also getting something similar 
![image](https://github.com/painebenjamin/app.enfugue.ai/assets/73911096/304130cf-8b10-4017-a55b-ce17818c2acd)

**Comment by coreshock** [comment]
@painebenjamin Can I fix the problem above without waiting for the next Windows release? Thanks!

**Comment by painebenjamin** [comment]
Hello! My apologies for the delay, I'm off on holiday this week.

Unfortunately I don't have any insight into what this error is, there is no specific usage of jaxlib in enfugue, it is a dependency of a dependency. There is some incompatibility with your environment that needs to be examined, I just need to figure out what is special about yours and OP's machines.

Could you potentially give me some more information about your environment? GPU and OS would be helpful, as well as your driver versions and anything else you're willing to volunteer.

As far as getting you up and going now, you can install via conda. Conda will analyze your machine and find a working version for your setup, it just takes a little more work to install. The documentation on [the latest release](https://github.com/painebenjamin/app.enfugue.ai/releases/tag/0.3.1) can walk you through it, but don't hesitate to ask for clarification if the documentation isn't explanatoryenough.

**Comment by painebenjamin** [comment]
@TrainerSnow that is very helpful, thank you.

Do you happen to have another global Python installation, by chance? We would be able to tell if you go into your environment variables and find a `PYTHONPATH` entry pointing somewhere other than your Anaconda installation.

To see environment variables on Windows, open the start menu and begin typing "Environment". You will see an option that says "Edit the system environment variables," click this. In the popup window, click "Environment Variables."

![image](https://github.com/painebenjamin/app.enfugue.ai/assets/57536852/7f16ce5d-f056-40f4-a765-de997154edc4)

**Comment by TrainerSnow** [comment]
Hi, this is the only env vars that i have:
![grafik](https://github.com/painebenjamin/app.enfugue.ai/assets/81151019/46c0b51c-e0e5-4e67-9ed6-5e8cac189c79)

**Comment by coreshock** [comment]
@painebenjamin @TrainerSnow Probably that was a question for me? :) Actually I removed PYTHONPATH system env variable.

When I try to locate available python installations, I'm getting multiple paths - probably this is the cause?
![image](https://github.com/painebenjamin/app.enfugue.ai/assets/73911096/79a5ff14-35ff-422e-ac3b-afc4a9be4c70)

**Comment by coreshock** [comment]
@painebenjamin I have to confess one stupid mistake: I tried to get **windows-cuda.yml** by right click + "save link as", which resulted in downloading the html instead of yml under the correct name. So the troubleshooting boils down only to environment installation via Conda via provided instructions. This part now runs okay, but jax still missing.

The best so far I could achieve is jax installation via **pip install jax[cuda111] -f https://whls.blob.core.windows.net/unstable/index.html --use-deprecated legacy-resolver**

But then I get this:

**(enfugue) F:\enfugue-server>pip show jax
Name: jax
Version: 0.4.6
Summary: Differentiate, compile, and transform Numpy code.
Home-page: https://github.com/google/jax
Author: JAX team
Author-email: jax-dev@google.com
License: Apache-2.0
Location: c:\programdata\anaconda3\envs\enfugue\lib\site-packages
Requires: numpy, opt-einsum, scipy
Required-by:

(enfugue) F:\enfugue-server>pip show jaxlib
WARNING: Package(s) not found: jaxlib**
